src/commands/add-organization-member.ts

```diff
@@ -0,0 +1,51 @@
+import {BaseCommand} from '../lib'
+import {getEnvironment} from '../lib/environments'
+import {flags} from '@oclif/command'
+
+const ADD_ORGANIZATION_MEMBER = `
+mutation AddOrganizationMember($member: AddOrgMember!) {
+    addOrganizationMember(input: $member) {
+      uid
+      name
+    }
+}
+`
+
+export default class AddOrganizationMember extends BaseCommand {
+  static description = 'Add a Member to an Organization'
+
+  static examples = [
+    '$ slash-graphql add-organization-member -o 0x123 -m test@gmail.com',
+  ]
+
+  static flags = {
+    ...BaseCommand.commonFlags,
+    member: flags.string({char: 'm', description: 'Member email ID', default: ''}),
```

Comment:
i'm sure there is a mandarory field. Why is default needed?

Reply:
in fact, if it's mandatory, make them params, not flags. See other commands that use params

Reply:
also, email address, not email id